### PR TITLE
feat(sql): implement function to convert two longs to UUID

### PR DESCRIPTION
**Pull request by jerrinot**
This PR introduces a new SQL function: `to_uuid(long, long)`. The function creates a UUID value out of 2 long values. 

Each long value has 64 bits. UUID values have 128 bits. The function combines bits from both longs to construct a UUID. The first long is interpreted as the lower 64 bits and the 2nd long is used as higher 64 bits of the resulting UUID value. 

Trivial example:
```sql
select to_uuid(2, 1) as uuid from long_sequence(1);
```
Expected result:
```
                uuid
00000000-0000-0001-0000-000000000002
```

Less trivial example:
```sql
create table x (lo long, hi long);
insert into x values (2, 1);
select to_uuid(lo, hi) as uuid from x;
```
Expected result:
```
                uuid
00000000-0000-0001-0000-000000000002
```

Expected use-case: Construct a unique ID out of 2 IDs received from different sources.

### CODE REVIEW

**Comment by ideoma** [comment]
#### [PR Coverage check]

:heart_eyes: **pass** : 19 / 19 (100.00%)




#### file detail

|   |path|covered line|new line|coverage|
|----|----|----|----|----|
|:large_blue_circle:|io/questdb/griffin/engine/functions/uuid/LongsToUuidFunctionFactory.java|19|19|100.00%|

**Comment by jerrinot** [comment]
documentation PR will follow upon merge (if that happens)

File: core/src/main/java/io/questdb/griffin/engine/functions/uuid/LongsToUuidFunctionFactory.java [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+        private final Function lo;
+
+        public LongsToUuidFunction(Function loLong, Function hiLong) {
+            this.lo = loLong;
+            this.hi = hiLong;
+        }
+
+        @Override
+        public void close() {
+            Misc.free(hi);
```
Comment (bluestreak01) [inline core/src/main/java/io/questdb/griffin/engine/functions/uuid/LongsToUuidFunctionFactory.java:69]: double-close

**Review by bluestreak01 — CHANGES_REQUESTED** [review]

**Comment by ideoma** [comment]
#### [PR Coverage check]

:heart_eyes: **pass** : 16 / 16 (100.00%)




#### file detail

|   |path|covered line|new line|coverage|
|----|----|----|----|----|
|:large_blue_circle:|io/questdb/griffin/engine/functions/uuid/LongsToUuidFunctionFactory.java|16|16|100.00%|

**Comment by ideoma** [comment]
#### [PR Coverage check]

:heart_eyes: **pass** : 16 / 16 (100.00%)




#### file detail

|   |path|covered line|new line|coverage|
|----|----|----|----|----|
|:large_blue_circle:|io/questdb/griffin/engine/functions/uuid/LongsToUuidFunctionFactory.java|16|16|100.00%|

**Pull request merged**